### ISSUE TITLE
Importing Perses dashboard is considering the project from metadata.project and not the input project

### ISSUE THREAD

**Issue by Nexucis** [perses/perses#1833]
### What did you do?

I tried to import a Perses dashboard using the UI 

### What did you expect to see?

I expected it works

### What did you see instead? Under which circumstances?

The system didn't consider the project where I wanted to import it. Instead it used the project coming from the metadata of the dashboard I'm importing (where I don't have the write permission)

![image](https://github.com/perses/perses/assets/4548045/31979c03-904d-46c8-8da7-8b697ee8b47a)

### Perses version

v0.44.0-rc2

### Perses configuration file

config from the online demo.

```json
{
  "security": {
    "readonly": false,
    "encryption_key": "<secret>",
    "enable_auth": true,
    "authorization": {
      "check_latest_update_interval": "30s",
      "guest_permissions": [
        {
          "actions": [
            "read"
          ],
          "scopes": [
            "*"
          ]
        },
        {
          "actions": [
            "create"
          ],
          "scopes": [
            "Project"
          ]
        }
      ]
    },
    "authentication": {
      "access_token_ttl": "15m",
      "refresh_token_ttl": "1d",
      "disable_sign_up": false,
      "providers": {
        "enable_native": true,
        "oauth": [
          {
            "slug_id": "github",
            "name": "Github",
            "client_id": "<secret>",
            "client_secret": "<secret>",
            "redirect_uri": null,
            "scopes": null,
            "disable_pkce": false,
            "auth_url": "https://github.com/login/oauth/authorize",
            "token_url": "https://github.com/login/oauth/access_token",
            "user_infos_url": "https://api.github.com/user",
            "custom_login_property": ""
          }
        ],
        "oidc": [
          {
            "slug_id": "google",
            "name": "Google",
            "client_id": "<secret>",
            "client_secret": "<secret>",
            "redirect_uri": null,
            "scopes": [
              "openid",
              "profile",
              "email"
            ],
            "disable_pkce": false,
            "issuer": "https://accounts.google.com",
            "discovery_url": null,
            "url_params": null
          }
        ]
      }
    }
  },
  "database": {
    "file": {
      "folder": "",
      "extension": "json",
      "case_sensitive": false
    }
  },
  "schemas": {
    "panels_path": "/etc/perses/cue/schemas/panels",
    "queries_path": "/etc/perses/cue/schemas/queries",
    "datasources_path": "/etc/perses/cue/schemas/datasources",
    "variables_path": "/etc/perses/cue/schemas/variables",
    "interval": "5m"
  },
  "important_dashboards": [
    {
      "project": "perses",
      "dashboard": "NodeExporterFull"
    }
  ],
  "information": "# Welcome to the Demo instance of Perses!\n\nThis public demo instance is here to show you what Perses looks like\nand what's possible to do with it. You can have a look at the example\ndashboards, or create your own ones!",
  "provisioning": {
    "folders": [
      "/etc/perses/provisioning"
    ],
    "interval": "1h"
  },
  "deactivate_front": false,
  "ephemeral_dashboards_cleanup_interval": "1d"
}
```

**Comment by Nexucis** [comment]
After investigating a bit, the issue is coming from the UI that doesn't replace `metadata.project` with the input `project`. No issue with the API call after that.

**Comment by Nexucis** [comment]
fixed with #1858
